### PR TITLE
Add documentation of location service endpoints to README

### PR DESCRIPTION
**Pull request by mpfilbin**
This comment extends the documentation in the README file with information about the user location endpoints.

Connects to https://github.com/codefordenver/fresh-food-connect/issues/19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,66 +55,255 @@ Any invalid request body parameters will result in a `422 Unprocessable Entity`
...
+        {
+            size: INT (none = 0 | small = 1 | medium = 2 | large = 3),
+            comments: String (max length 255 characters)
+            
+        }
     
  Response:
-    HTTP 204 NO CONTENT
+ 
+        HTTP 204 NO CONTENT
```
Comment (switzersc) [inline README.md]: responses to PUTs are always the representation of the resource (basically the same json body as the POST create response), not no content.

**Comment by switzersc** [comment]
Other than the comment on PUT responses, this looks awesome to me :)

File: README.md (hunk 1 above)
Comment (mpfilbin) [inline README.md]: Updated :+1:

**Comment by wdoug** [comment]
:shipit:

**Pull request merged**